dev/TeachingTip/TeachingTip_rs1_themeresources.xaml

```diff
@@ -4,22 +4,28 @@
     xmlns:local="using:Microsoft.UI.Xaml.Controls">
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
-            <SolidColorBrush x:Key="TeachingTipBorderBrush" Color="#5C000000"/>
+            <StaticResource x:Key="TeachingTipBorderBrush" ResourceKey="SystemControlTransientBorderBrush"/>
```

Comment:
Semantically this change feels correct. But the value is being modified from #5C000000 to #FF767676 (in dark theme). Has anyone validated that this still looks correct?

Reply:
In generic.xaml I see this value as:
```xaml
            <SolidColorBrush x:Key="SystemControlTransientBorderBrush" Color="#000000" Opacity="0.36" />
```

Which is .36 * 255 = 92 = 0x5C. So I *think* it should be the same? Same for the light theme -- it's .14 in generic.xaml which comes out to ~0x24.